synfig-studio/src/gui/pluginmanager.h

```diff
@@ -30,6 +30,8 @@
 
 /* === H E A D E R S ======================================================= */
 
+#include <glibmm/refptr.h>
```

Comment:
`glibmm/refptr.h` - is unused, so better to remove it.
`gtkmm/application.h` - is unused too, you need `#include <sigc++/signal.h>` instead for `sigc::signal`